configuration_controller/request_consumer/request_db_consumer.py

```diff
@@ -10,16 +12,21 @@
 class RequestDBConsumer:
     """Class to consume requests from the db"""
 
-    def __init__(self, request_type: str):
+    def __init__(self, request_type: str, request_max_batch_size: int):
         self.request_type = request_type
+        self.request_max_batch_size = request_max_batch_size
```

Comment:
We could add 2 things here:
1. `0` makes little sense so in such case maybe throw an exception
2. A negative value could turn on "no limit" mode where the consumer could simply try to get all pending requests